vscode/webviews/components/mcp/ServerHome.tsx

```diff
@@ -178,21 +178,24 @@ export function ServerHome({ mcpServers }: ServerHomeProps) {
             className="tw-flex tw-flex-col tw-h-full tw-py-4 tw-bg-transparent tw-px-2 tw-mb-4 tw-overscroll-auto"
             disablePointerSelection={true}
         >
-            <header className="tw-inline-flex tw-mt-4 tw-px-4 tw-gap-4">
+            <header className="tw-flex tw-items-center tw-justify-between tw-mt-4 tw-px-4">
+                <div className="tw-flex tw-items-center tw-font-semibold tw-text-lg">
+                    <ServerIcon size={16} className="tw-mr-3" /> MCP Servers
+                </div>
                 <Button
-                    variant="secondary"
-                    className="tw-bg-popover tw-border tw-border-border !tw-justify-between"
+                    variant="ghost"
+                    size="icon"
+                    className="tw-h-8"
                     onClick={() =>
                         getVSCodeAPI().postMessage({
                             command: 'command',
                             id: 'workbench.action.openSettings',
                             arg: '@ext:sourcegraph.cody-ai cody.mcpServers',
                         })
                     }
+                    title="Server configuration settings"
```

Comment:
```suggestion
                    title="View JSON"
```

@alexromano like this?

Reply:
does this change the button from icon to text or just the tooltip? i was thinking we'd make it a full button that says "View JSON"

Reply:
like this? 
![image](https://github.com/user-attachments/assets/1268d877-7b7e-4d43-a049-f07bfad7af50)

Reply:
yeah but where the gear icon was before

Reply:
Updated